macros/parsers/parserRadioMultiAnswer.pl

```diff
@@ -288,8 +288,10 @@ sub new {
 		size              => undef,
 		checked           => undef,
 		uncheckable       => 0,
+		showInStatic      => 1,
 		@inputs
 	);
+	$options{originalLabels} = $options{labels};
```

Comment:
This line isn't needed.  Line 600 below takes care of setting the `originalLabels` for the `$self` object.  This value might not be the correct value anyway, since the label option could be set using the `with` method at a later time.

Reply:
I think I hit an error though, because this might never get set by line 600, if line 598 executes. And then this is referenced at line 719.

But what's the best way to do this? I want to capture if the problem author intends for 123, ABC, etc labeling and I was hoping to avoid actually inspecting the array of labels.

Reply:
I see.  So in the case that the author passes an array of labels to begin with (and there are enough labels), then line 600 won't execute.  So just move line 600 to before line 598, and make it `$self->{originalLabels} //= $self->{labels};`.  Then the `//=` will only set `$self->{originalLabels}` the first time since it will be defined for all later calls.